### PR TITLE
Remove 1.22 from docs

### PR DESCRIPTION
**Pull request by zafs23**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,15 +85,15 @@ including critical vulnerability fixes, to EKS-D v1.22 once support ends.
 **Due to the increased security risk this poses, it is HIGHLY recommended that 
 users of v1.22 update to a supported version (v1.23+) as soon as possible.**
 
-### Kubernetes 1.18, 1.19, 1.20, and 1.21: DEPRECATED
+### Kubernetes 1.18, 1.19, 1.20, and 1.22: DEPRECATED
```
Comment (markapruett) [inline README.md:88]: Leave 1.21 in there but also add 1.22

File: README.md [hunk 2]
```diff
@@ -87,15 +85,15 @@ including critical vulnerability fixes, to EKS-D v1.22 once support ends.
...
 
 In alignment with the [Amazon EKS release calendar](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar),
 EKS Distro has discontinued support of Kubernetes v1.18 - v1.21. While there are
```
Comment (markapruett) [inline README.md:91]:
```suggestion
EKS Distro has discontinued support of Kubernetes v1.18 - v1.22. While there are
```

**Comment by markapruett** [comment]
Several small issues. Make sure to take another pass to ensure we don't miss anything.

File: README.md [hunk 3]
```diff
@@ -1,9 +1,13 @@
 ## EKS Distro Repository
 
+### 🚨🚨🚨 IMPORTANT INFORMATION ABOUT 1.22 SUPPORT 🚨🚨🚨
```
Comment (kschumy) [inline README.md:3]: Do we need this section here? It’s been here for several weeks.

**Comment by kschumy** [comment]
We need to remove it from supported release branches

File: README.md (hunk 3 above)
Comment (zafs23) [reply README.md:3]: @kschumy I thought we are announcing that we have stopped supporting 1.22, but if we are thinking SNS as the announcement, I can get rid of it.

**Pull request merged**